src/devices/efth800.c

```diff
@@ -41,6 +41,8 @@ Data layout:
 The sensor sends messages at intervals of about 57-58 seconds.
 */
 
+#include <stdlib.h>
```

Comment:
What for? Please remove.

Reply:
That was left from an earlier version of the code. I've removed it.